echo.go

```diff
@@ -0,0 +1,17 @@
+package testy
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+// AddEchoRoutes adds routes to an Echo router that can run tests and retrieve tests results.
+func AddEchoRoutes(router *echo.Group) {
```

Comment:
the idea is that eventually we'll have a variety of endpoints here to run specific tests and get stats updates on in-progress tests